ElectronNET.API/App.cs

```diff
@@ -495,9 +586,13 @@ public string Name
         public Task<string> GetNameAsync() => BridgeConnector.OnResult<string>("appGetName", "appGetNameCompleted");
 
 
-        internal App() 
+        private App()
         {
-            CommandLine = new CommandLine();
+            if (OperatingSystem.IsMacOS() || OperatingSystem.IsLinux())
+            {
+                AppContext.SetSwitch("System.Drawing.EnableUnixSupport", true);
```

Comment:
this is no longer required with the move away from `System.Drawing.Common` done in PR #658